push/model/src/main/java/org/jboss/aerogear/unifiedpush/message/InternalUnifiedPushMessage.java

```diff
@@ -0,0 +1,26 @@
+package org.jboss.aerogear.unifiedpush.message;
```

Comment:
I wonder if we should move this class out of the push/model package.

That API is used on the Java Sender, and there it does not make sense to have these extra information (like IP address, and others).

Perhaps it's good to move it to push/sender module?